programs/vote/src/vote_processor.rs

```diff
@@ -152,6 +152,15 @@ declare_process_instruction!(Entrypoint, DEFAULT_COMPUTE_UNITS, |invoke_context|
             )
         }
         VoteInstruction::Vote(vote) | VoteInstruction::VoteSwitch(vote, _) => {
+            if invoke_context
+                .feature_set
+                .is_active(&feature_set::deprecate_legacy_vote_ixs::id())
+                && invoke_context
+                    .feature_set
+                    .is_active(&feature_set::enable_tower_sync_ix::id())
```

Comment:
do we need to wait for the separate vote instruction enable tower to deprecate this?

Reply:
nope, this is just a precaution in case I get hacked or something. don't want to end up in a situation where all vote ixs are disabled.